app/legacy_lib/update_donation.rb

```diff
@@ -172,7 +172,7 @@ def self.correct_donations_when_date_and_payments_are_off(id)
 
       donation.save!
 
-      return @payments_corrected
+      @payments_corrected
```

Comment:
I don't even know what this particular method does. It's untested but it's not used.